### PR TITLE
Reading packages by GUI and bus implementation

### PR DESCRIPTION
**Pull request by pawlusha**
MessageBus (logic->GUI) implemented.
Received packages are now properly read by GUI

### CODE REVIEW

File: src/main/java/com/fourmusketeers/client/view/GameWindowView.java [hunk 1]
```diff
@@ -100,4 +115,44 @@ private VBox getSpacingBar() {
...
+                enemyButtons.forEach((button) -> button.getButton().setDisable(false));
+            }
+            opponentPlayer.setText(whoStarts.getOpponentName());
+        });
+    }
+
+    private void updateBoard(GameTurnInfoToClient infoPackage) {
+        Platform.runLater(() -> {
+            Integer shotPosition = infoPackage.getDetonationPosition().get(0);
+
```
Comment (arturAR) [inline src/main/java/com/fourmusketeers/client/view/GameWindowView.java]: 👃

File: src/main/java/com/fourmusketeers/client/bus/EventBus.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    @Override
+    public void run() {
+        while (!finished) {
+            pushEventsToObservers();
+        }
+    }
+
+}
```
Comment (yakisoba0) [inline src/main/java/com/fourmusketeers/client/bus/EventBus.java]: 👍

File: src/main/java/com/fourmusketeers/client/bus/Observable.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package com.fourmusketeers.client.bus;
+
+/**
+ * @author arek
+ * @since 31.07.17
+ */
+public abstract class Observable {
+
```
Comment (yakisoba0) [inline src/main/java/com/fourmusketeers/client/bus/Observable.java]: "Observable"? It is an Abstract class and has implementation so maybe "AbstractObserver" or something?

File: src/main/java/com/fourmusketeers/client/logic/ClientGame.java [hunk 4]
```diff
@@ -44,4 +67,8 @@ private void runClientGame() {
     public void run() {
         runClientGame();
     }
```
Comment (yakisoba0) [inline src/main/java/com/fourmusketeers/client/logic/ClientGame.java]: In  my opinion this methods don't need to be separetated

**Pull request merged**